go/oasis-node/cmd/common/pprof/pprof.go

```diff
@@ -79,7 +75,11 @@ func (p *pprofService) Start() error {
 
 	go func() {
 		if err := p.server.Serve(p.listener); err != nil {
-			p.errCh <- err
```

Comment:
If we tried to stop the service, the server was closed, this channel blocked and the service was never stopped.